### PR TITLE
[move source language] Fixed formatting for long string literals

### PR DESCRIPTION
**Pull request by tnowacki**
## Motivation

- Fixed long string literals so rustfmt/cargo fmt would run

## Test Plan

- eyes
- cargo fmt
- cargo test

### CODE REVIEW

**Comment by sunshowers** [comment]
uhh a comment got lost

File: language/move-lang/rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+edition = "2018"
+use_field_init_shorthand = true
```
Comment (sunshowers) [inline language/move-lang/rustfmt.toml]: Should this be in the top-level rustfmt.toml?

Comment (tnowacki) [reply language/move-lang/rustfmt.toml]: Ah, I forgot to remove this file. It was just there to turn on experimental features so I could find the spots to fix :) 

Will remove before landing

**Pull request closed without merging**